### DISCUSSION TITLE
Alpine LXC - Internet check fail

### DISCUSSION THREAD

**Discussion by WurstKarate** [BassT23/Proxmox#90] [Q&A] [answered]
Hi! I am trying to update an alpin-linux lxc and got this message

![image](https://github.com/BassT23/Proxmox/assets/81300762/001fd3cd-8c47-4328-9002-235ee35b5293)

I would be thankful for any help and realy great work by the way!

**Comment by BassT23** [comment]
are you sure the lxc has internet connection?

I don't use alpine, only for testing. So the first line dan't say me anything.

In the second line; the script check, if the container/VM is online to run updates.

please try out ``ping google.com`` on the lxc

and ``pct exec <CONTAINER ID> -- bash -c "ping -q -c1 $CHECK_URL`` on the host

**Comment by BassT23** [comment]
ok, command from host bring me the same error, ...

```
root@elitedesk:~# pct exec 194 -- bash -c "ping -q -c1 google.com"
lxc-attach: 194: ../src/lxc/attach.c: lxc_attach_run_command: 1841 No such file or directory - Failed to exec "bash"
```

I need to exlude alpine or change the command

Will have a look on this in the next few days. Till I fixed it, you could use v3.8.5

**Reply by WurstKarate** [reply]
Thank you very much!

**Reply by BassT23** [reply, marked as answer]
https://github.com/BassT23/Proxmox/issues/91

**Comment by BassT23** [comment]
v3.9.1 (beta)

fixed - till i find a better solution, the script skip internet check for alpine ;)

To update to latest beta use ``update beta -up``
